supervisord/tox.ini

```diff
@@ -14,16 +14,8 @@ deps =
     -rrequirements-dev.txt
 commands =
     pip install -r requirements.in
-    pytest -v -m integration
-
-[testenv:unit]
-platform = linux|darwin|win32
-deps =
-    -e../datadog_checks_base[deps]
-    -rrequirements-dev.txt
-commands =
-    pip install -r requirements.in
-    pytest -v -m unit
+    integration_3.3.3: pytest -v -m integration
```

Comment:
could this just be `integration` for now to clean up the name?

Reply:
true but it's ok :)